src/components/Details/Details.css

```diff
@@ -0,0 +1,21 @@
+summary {
+    list-style: none;
+}
+
+/* Safari */
+
+summary ::-webkit-details-marker {
+    list-style: none;
+}
+
+details[open] .faIcon{
+    transform: rotate(180deg);
+}
+
+details[open] {
+    border-top: 0.2em solid black ;
+}
```

Comment:
Here you can use CSS nesting, so you don't duplicate the `details[open]` selector.